includes/test.php

```diff
@@ -0,0 +1,6 @@
+<?php
```

Comment:
@bartboy011 This could be removed.

Reply:
@cristiangrama Good catch, that was an accidental commit